### PR TITLE
refactor: use String#repeat

### PR DESCRIPTION
**Pull request by MartinWitt**
For this refactoring i`m unsure if it's better readable. Some opinions?

### CODE REVIEW

File: src/main/java/spoon/reflect/visitor/PrinterHelper.java [hunk 1]
```diff
@@ -144,9 +144,7 @@ private void writeTabsInternal() {
 				if (env != null) {
 					indentationSize = env.getTabulationSize();
 				}
-				for (int j = 0; j < indentationSize; j++) {
-					sbf.append(' ');
-				}
+				sbf.append(Character.toString(' ').repeat(indentationSize));
```
Comment (algomaster99) [inline src/main/java/spoon/reflect/visitor/PrinterHelper.java:147]: This refactoring is definitely better than `for-loop`. I would suggest we can further shorten this to:
```java
sb.append("\s".repeat(indentationSize));
```

I am unsure if `\s` escape sequence works like `\t` does.

**Review by algomaster99 — CHANGES_REQUESTED** [review]

File: src/main/java/spoon/reflect/visitor/PrinterHelper.java (hunk 1 above)
Comment (monperrus) [inline src/main/java/spoon/reflect/visitor/PrinterHelper.java:147]:
```suggestion
				sbf.append(" ".repeat(indentationSize));
```

Comment (MartinWitt) [reply src/main/java/spoon/reflect/visitor/PrinterHelper.java]: I merge @monperrus suggestions, but yours looks fine too.

**Review by algomaster99 — APPROVED** [review]
